cpu/atmega_common/Makefile.include

```diff
@@ -6,4 +6,7 @@ export INCLUDES += -I$(RIOTCPU)/atmega_common/include \
 # avr libc needs some RIOT-specific support code
 USEMODULE += avr_libc_extra
 
+PSEUDOMODULES += atmega_pcint
+PSEUDOMODULES += atmega_pcint%
```

Comment:
Just add

```
ifneq (,$(filter atmega_pcint,$(USEMODULE))
    USEMODULE += atmega_pcint0 atmega_pcint1 atmega_pcint2 atmega_pcint3
endif
```

And you no longer need to handle `MODULE_ATMEGA_PCINT` in c code.

Reply:
See comment above. Will try.

Reply:
This should be moved to `makesfiles/pseudomodules.inc.mk`